### ISSUE TITLE
Spacing of small version of star rating

### ISSUE THREAD

**Issue by MewenLeHo** [Orange-OpenSource/Orange-Boosted-Bootstrap#2072]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Following the work on #2035, an a11y was spotted:

The hitbox of small stars is too small: actually `20*20px`, should be at least `24*24px` (hitbox of large stars is `25*25px` 🟢)

### Reduced test cases

First thing to try will be to increase the spacing between two stars to make a larher clickable zone without changing the size of the small stars.

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Firefox, Microsoft Edge

### What version of Boosted are you using?

v5.2.3 (current main branch)

**Comment by MaxLardenois** [comment]
Boosted only, won't fix
